docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -727,6 +727,23 @@ Your new custom cloud is ready to use only after its status changes to
      --display-name "CUSTOM_CLOUD_DISPLAY_NAME"
    ```
 
+   where
+
+   - `organization-id` is the identifier of your Aiven organization that is to be
+     connected with your own cloud account to create the custom cloud.
+   - `deployment-model` is the type of [network architecture](/docs/platform/concepts/byoc#byoc-deployment)
+     your custom cloud uses:
+     - `public` model allows the Aiven control plane to connect to the service nodes via the
+       public Internet.
+     - `private` model routes traffic through a proxy for additional security utilizing a
+       bastion host physically separated from the Aiven services.
```

Comment:
I think `DEPLOYMENT_MODEL_NAME` is one of `standard` and `standard_public`. Not `private` or `public`